ingestion_tools/scripts/data_validation/standardized/tests/test_frame.py

```diff
@@ -1,9 +1,31 @@
+from typing import Dict, List, Union
 
+import allure
 import pytest
+import tifffile
 from data_validation.shared.helper.twodee_helper import FrameTestHelper
+from mrcfile.mrcinterpreter import MrcInterpreter
 
 
 @pytest.mark.frame
 @pytest.mark.parametrize("dataset, run_name", pytest.cryoet.dataset_run_combinations, scope="session")
 class TestFrame(FrameTestHelper):
-    pass
+    @allure.title("Frames: When isGainCorrected == False, a Gains entity exists for the run")
+    def test_is_gain_corrected_false(self,
+                         frame_metadata: Dict,
+                         gain_headers: Dict[str, Union[List[tifffile.TiffPage], MrcInterpreter]], # this is skipped if it is not found
+                         ):
+        if frame_metadata.get("is_gain_corrected") is False: # todo is none considered false?
```

Comment:
I would treat `None` as `False` for this case.

Reply:
By that you mean the `if` condition should be true, if  `is_gain_corrected` is `None`, which would be equivalent to 
```suggestion
        if frame_metadata.get("is_gain_corrected", False) is False:
```